src/SignalR/common/Http.Connections/src/Features/IHttpContextFeature.cs

```diff
@@ -1,15 +1,23 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
 using System.Collections.Generic;
 using System.Text;
+using Microsoft.AspNetCore.Connections;
 using Microsoft.AspNetCore.Http;
 
 namespace Microsoft.AspNetCore.Http.Connections.Features
 {
+    /// <summary>
+    /// Feature set on the <see cref="ConnectionContext"/> that provides access to the <see cref="Http.HttpContext"/>
+    /// associated with the connection if there is one.
+    /// </summary>
     public interface IHttpContextFeature
     {
+        /// <summary>
+        /// The <see cref="Http.HttpContext"/> associated with the connection.
+        /// </summary>
```

Comment:
```suggestion
        /// </summary>
        /// <remarks>
        /// SignalR connections can run on top of HTTP transports like WebSockets or Long Polling, or other non-HTTP transports. As a result,
        /// this API can sometimes return <see langword="null"/> depending on the configuration of your application.
        /// </remarks>
```

Reply:
Removed SignalR since this layer knows nothing about SignalR.